accounts-journey/src/androidTest/java/app_common/GenericInteractions.kt

```diff
@@ -0,0 +1,40 @@
+package app_common
+
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+
+fun ViewInteraction.shouldBeDisplayed(): ViewInteraction =
```

Comment:
We can simplify code by adding static imports for `ViewAssertions` and `ViewMatchers`.

It allows you to use `matches(withText(stringResource))` instead of `ViewAssertions.matches(ViewMatchers.withText(stringResource))`. 

Please apply these changes for the whole file

Reply:
Fixed for the whole file